backend/app/treasury/tests/adapters/covalent/test_covalent_pricefeed.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=protected-access
+from json import loads
+
+from dateutil.parser import parse
+from dateutil.tz import UTC
+from fakeredis import FakeRedis
+from pytest import MonkeyPatch, mark
+
+from backend.app.treasury.adapters import covalent_pricefeed
+
+from .conftest import covalent_hist_prices_v2_transfers, mocked_datetime
+
+
+class FakeCovalentResponse:
```

Comment:
As suggested in my branch `fix/pagination`, you could rather use `mock.Mock(spec=httpx.Response)` which I believe is simpler and allows useful features like stubs (`side_effect`). But I'm 100% fine with this version for now.